src/reforms/tier_01/1.txt

```diff
@@ -29,6 +29,9 @@ governments:theocracy:reform_levels:1 += new tier_1_forelhost_priestdom = {
 		immortal_ruler = yes
 		rahgot_ruler = yes
 	}
+	potential = {
+		has_reform = tier_1_forelhost_priestdom
```

Comment:
why u added potential?

Reply:
so you don't get all of them visible at the same time, it was bugging me a lot.

Reply:
oh great ,that was bothering me too